dbms/src/Parsers/ASTTTLElement.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <Parsers/IAST.h>
+
+
+namespace DB
+{
+/** Element of TTL expression.
+  */
+class ASTTTLElement : public IAST
+{
+public:
+    enum DestinationType
```

Comment:
`MoveDestinationType` was more clear.

Reply:
Why it's here? It used not only with TTLs. Let's make it separate `enum class`,

Reply:
.